File: POO/EJ1JAVA/Genero.java
package POO.EJ1JAVA;

public class Genero {
    private String nombre;
    private int edadMinimaRecomendada;

    public Genero(String pnombre, int pedadMinimaRecomendada) {
        this.nombre = pnombre;
        this.edadMinimaRecomendada = pedadMinimaRecomendada;
    }

    public String getNombre() {
        return this.nombre;
    }

    public void setNombre(String pnombre) {
        this.nombre = pnombre;
    }

    public int getEdadMinimaRecomendada() {
        return this.edadMinimaRecomendada;
    }

    public void setEdadMinimaRecomendada(int pedadMinimaRecomendada) {
        this.edadMinimaRecomendada = pedadMinimaRecomendada;
    }
}

File: POO/EJ1PY/Genero.py
class Genero:
    # Atributos
    nombre = ""
    edad = 0

    #Método constructor
    def init(self,pnombre,pedad):
        self.nombre = pnombre
        self.edad = pedad

#Getters y setters
    def getNombre(self):
        return self.nombre

    def setNombre(self,pnombre):
        self.nombre = pnombre

    def getEdad(self):
        return self.edad

    def setEdad(self,pedad):
        self.edad = pedad